StortSpelProjekt/Engine/src/Misc/Window.cpp

```diff
@@ -138,6 +141,25 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
 
 				SetCursorPos(500, 400);
 			}
+
+			// This is temporarly to make sure that a mouse click works even though the 'alt' key is pressed
+			if (DEVELOPERMODE_DEVINTERFACE == false || Input::GetInstance().GetKeyState(SCAN_CODES::ALT))
+			{
+				auto inputData = raw->data.mouse;
+				MOUSE_BUTTON button = static_cast<MOUSE_BUTTON>(inputData.usButtonFlags);
+
+				switch (button)
+				{
+				case MOUSE_BUTTON::LEFT_DOWN:
+					Input::GetInstance().SetMouseButtonState(button, true);
+					break;
+				case MOUSE_BUTTON::LEFT_UP:
+					button = static_cast<MOUSE_BUTTON>(static_cast<int>(button) / 2);
+					Input::GetInstance().SetMouseButtonState(button, false);
+				default:
+					break;
+				}
+			}
```

Comment:
Just a comment: Should this be here when you merge to develop? Is this to prevent when the mouse is "Lost" when the user pressed "alt-enter" to go fullscreen?

Reply:
This makes it possible to register a click even though you are pressing alt